### PR TITLE
adding realtime update to all server-side sdk docs

### PR DESCRIPTION
**Pull request by cdruxerman**

### CODE REVIEW

File: docs/sdk/server-side-sdks/java/java-usage.md [hunk 1]
```diff
@@ -186,3 +186,33 @@ This will send a request to our EdgeDB API to save the custom data under the use
...
+import com.devcycle.sdk.server.local.api.DevCycleLocalClient;
+import com.devcycle.sdk.server.local.model.DevCycleLocalOptions;
+
+public class MyClass {
+
+    private DevCycleLocalClient client;
+
+    public MyClass() {
+        DevCycleLocalOptions options = DevCycleLocalOptions.builder()
+            .enableBetaRealtimeUpdates(true)
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/java/java-usage.md:212]: Java uses a builder state - https://github.com/DevCycleHQ/java-server-sdk/blob/main/src/main/java/com/devcycle/sdk/server/local/model/DevCycleLocalOptions.java

File: docs/sdk/server-side-sdks/java/java-gettingstarted.md [hunk 2]
```diff
@@ -73,3 +62,20 @@ public class MyClass {
...
+| disableAutomaticEventLogging | Boolean        | Disables logging of sdk generated events (e.g. aggVariableEvaluated, aggVariableDefaulted) to DevCycle.                                                                      |
+| disableCustomEventLogging    | Boolean        | Disables logging of custom events, from `track()` method, and user data to DevCycle.                                                                                         |
+| flushEventQueueSize          | Number         | Controls the maximum size the event queue can grow to until a flush is forced. Defaults to `1000`.                                                                           |
+| maxEventQueueSize            | Number         | Controls the maximum size the event queue can grow to until events are dropped. Defaults to `2000`.                                                                          |
+| apiProxyURL                  | String         | Allows the SDK to communicate with a proxy of DevCycle bucketing API / client SDK API.                                                                                       |
```
[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/java/java-gettingstarted.md:80]: https://github.com/DevCycleHQ/go-server-sdk/blob/main/configuration.go#L22

File: docs/sdk/server-side-sdks/dotnet/dotnet-usage.md [hunk 3]
```diff
@@ -107,3 +107,30 @@ To further enable various deployment configurations - we provide a background pr
...
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
+
+To enable Beta Realtime Updates, pass in the `enableBetaRealtimeUpdates` option to the SDK initialization:
+
+```csharp
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/dotnet/dotnet-usage.md:123]: https://github.com/DevCycleHQ/dotnet-server-sdk/blob/main/DevCycle.SDK.Server.Common/Model/Local/DevCycleLocalOptions.cs

File: docs/sdk/server-side-sdks/php/php-usage.md [hunk 4]
```diff
@@ -195,3 +195,26 @@ To run the tests, use:
...
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
+
+To enable Beta Realtime Updates, pass in the `enableBetaRealtimeUpdates` option to the SDK initialization:
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/php/php-usage.md:208]: https://github.com/DevCycleHQ/php-server-sdk/blob/main/lib/Model/DevCycleOptions.php

File: docs/sdk/server-side-sdks/python/python-gettingstarted.md [hunk 5]
```diff
@@ -58,3 +58,33 @@ user = DevCycleUser(
...
+from devcycle_python_sdk import DevCycleCloudClient, DevCycleCloudOptions
+from devcycle_python_sdk.models.user import DevCycleUser
+import os
+
+options = DevCycleCloudOptions(enable_edge_db=True)
+
+devcycle_client = DevCycleCloudClient(os.environ["DEVCYCLE_SERVER_SDK_KEY"], options)
+```
+
+| DevCycle Option              | Type           | Description                                                                                                                                                                  |
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/python/python-gettingstarted.md:76]: https://github.com/DevCycleHQ/python-server-sdk/blob/main/devcycle_python_sdk/options.py

File: docs/sdk/server-side-sdks/ruby/ruby-usage.md [hunk 6]
```diff
@@ -163,3 +163,28 @@ This will send a request to our EdgeDB API to save the custom data under the use
...
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
+
+To enable Beta Realtime Updates, pass in the `enableBetaRealtimeUpdates` option to the SDK initialization:
+
+```ruby
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/ruby/ruby-usage.md:178]: https://github.com/DevCycleHQ/ruby-server-sdk/blob/main/lib/devcycle-ruby-server-sdk/localbucketing/options.rb

**Review by JamieSinn — CHANGES_REQUESTED** [review]

File: docs/sdk/server-side-sdks/java/java-gettingstarted.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/java/java-gettingstarted.md:80]: @JamieSinn is it right that you linked go here for the java sdk options?

[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/java/java-gettingstarted.md:80]: nope - mis-link

[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/java/java-gettingstarted.md:80]: https://github.com/DevCycleHQ/java-server-sdk/blob/main/src/main/java/com/devcycle/sdk/server/local/model/DevCycleLocalOptions.java

File: docs/sdk/server-side-sdks/go/go-gettingstarted.md [hunk 7]
```diff
@@ -74,3 +68,55 @@ log.Println("DevCycle client not guaranteed to be initialized yet")
...
+|------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| logger                       | DevCycleLogger | Logger override to replace default logger                                                                                                                                    |
+| logLevel                     | String         | Set log level of the default logger. Options are: `debug`, `info`, `warn`, `error`. Defaults to `info`.                                                                      |
+| enableCloudBucketing         | Boolean        | Switches the SDK to use Cloud Bucketing (via the DevCycle Bucketing API) instead of Local Bucketing.                                                                         |
+| enableEdgeDB                 | Boolean        | Enables the usage of EdgeDB for DevCycle that syncs User Data to DevCycle. <br />NOTE: This is only available with Cloud Bucketing.                                          |
```
[the remark this answers is not in the document]
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:112]: https://github.com/DevCycleHQ/go-server-sdk/blob/main/configuration.go#L22

File: docs/sdk/server-side-sdks/dotnet/dotnet-usage.md [hunk 8]
```diff
@@ -107,3 +107,30 @@ To further enable various deployment configurations - we provide a background pr
...
+    client = new DevCycleLocalClientBuilder()
+        .SetSDKKey("<DEVCYCLE_SERVER_SDK_KEY>")
+        .SetOptions(new DevCycleLocalOptions(enableBetaRealtimeUpdates: true, eventFlushIntervalMs: 60000))
+        .SetInitializedSubscriber((o, e) => {
+            if (e.Success) {
+                ClientInitialized();
+            } else {
+                Console.WriteLine($"Client did not initialize. Errors: {e.Errors}");
+            }
+        })
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/dotnet/dotnet-usage.md:133]: I would remove this if we're not fully specifying all the data/functions it calls.

File: docs/sdk/server-side-sdks/go/go-gettingstarted.md [hunk 9]
```diff
@@ -74,3 +68,57 @@ log.Println("DevCycle client not guaranteed to be initialized yet")
...
+
+func main() {
+	sdkKey := os.Getenv("DEVCYCLE_SERVER_SDK_KEY")
+
+	options := devcycle.Options{
+		EnableEdgeDB:                 false,
+		EnableCloudBucketing:         false,
+		EventFlushIntervalMS:         30 * time.Second,
+		ConfigPollingIntervalMS:      1 * time.Minute,
+		RequestTimeout:               30 * time.Second,
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/go/go-gettingstarted.md:94]: These are defaults - remove?

File: docs/sdk/server-side-sdks/dotnet/dotnet-gettingstarted.md [hunk 10]
```diff
@@ -87,17 +84,63 @@ namespace Example {
...
+    public class Example {
+        private static DevCycleLocalClient client;
+
+        static async Task Main(string[] args) {
+            client = new DevCycleLocalClientBuilder()
+                .SetSDKKey("<DEVCYCLE_SERVER_SDK_KEY>")
+                .SetOptions(new DevCycleLocalOptions(configPollingIntervalMs: 60000, eventFlushIntervalMs: 60000))
+                .SetInitializedSubscriber((o, e) => {
+                    
+                })
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/dotnet/dotnet-gettingstarted.md:116]: This is no-op - remove?

File: docs/sdk/server-side-sdks/php/php-gettingstarted.md [hunk 11]
```diff
@@ -37,3 +37,24 @@ try {
 }
 ```
 [//]: # (wizard-initialize-end)
+
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when registering the DevCycleModule:
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/php/php-gettingstarted.md:43]: DevCycleModule doesn't exist - copy pasta?

File: docs/sdk/server-side-sdks/php/php-usage.md [hunk 12]
```diff
@@ -195,3 +195,26 @@ To run the tests, use:
...
+$options = new DevCycleOptions({
+    enableBetaRealtimeUpdates: true
+  });
+$devCycleClient = new DevCycleClient(
+    sdkKey: getenv("DEVCYCLE_SERVER_SDK_KEY"),
+    dvcOptions: $options);
+$user_data = new DevCycleUser(array(
+  "user_id"=>"my-user"
+));
+```
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/php/php-usage.md:220]: None of this is accurate. PHP does not have this functionality.

File: docs/sdk/server-side-sdks/python/python-usage.md [hunk 13]
```diff
@@ -171,3 +171,24 @@ To further enable various deployment configurations - we provide a background pr
...
+
+```python
+from devcycle_python_sdk import DevCycleCloudClient, DevCycleCloudOptions
+from devcycle_python_sdk.models.user import DevCycleUser
+import os
+
+options = DevCycleCloudOptions(enable_beta_realtime_updates=True)
+
+devcycle_client = DevCycleCloudClient(os.environ["DEVCYCLE_SERVER_SDK_KEY"], options)
+```
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/python/python-usage.md:194]: You're passing in the cloud client here - which doesn't have functionality for realtime updates

File: docs/sdk/server-side-sdks/go/go-gettingstarted.md [hunk 14]
```diff
@@ -74,3 +68,57 @@ log.Println("DevCycle client not guaranteed to be initialized yet")
...
+
+	devcycle "github.com/devcyclehq/go-server-sdk/v2"
+)
+
+func main() {
+	sdkKey := os.Getenv("DEVCYCLE_SERVER_SDK_KEY")
+
+	options := devcycle.Options{
+		EnableEdgeDB:                 false,
+		EnableCloudBucketing:         false,
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/go/go-gettingstarted.md:91]: These are defaults - remove?

File: docs/sdk/server-side-sdks/java/java-gettingstarted.md [hunk 15]
```diff
@@ -73,3 +62,31 @@ public class MyClass {
...
+
+### Local Bucketing Options
+
+| DevCycle Option              | Type           | Description                                                                                                                                                                  |
+|------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| customLogger                       | IDevCycleLogger | Logger override to replace default logger                                                                                                                                    |
+| restOptions                       | IRestOptions | insert description                                                                                                                                    |
+| configPollingIntervalMS      | Int         | Controls the polling interval in milliseconds to fetch new environment config changes, defaults to 30 seconds, minimum value is 1 second.                                    |
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/java/java-gettingstarted.md:72]: `int` should not be capitalized

File: docs/sdk/server-side-sdks/java/java-gettingstarted.md [hunk 16]
```diff
@@ -73,3 +62,31 @@ public class MyClass {
...
+| eventFlushIntervalMS         | Int         | Controls the interval between flushing events to the DevCycle servers, defaults to 10 seconds.                                                                               |
+| flushEventQueueSize          | Int         | Controls the maximum size the event queue can grow to until a flush is forced. Defaults to `1000`.                                                                           |
+| maxEventQueueSize            | Int         | Controls the maximum size the event queue can grow to until events are dropped. Defaults to `2000`.                                                                          |
+| eventRequestChunkSize            | Int         | Insert Description. Defaults to `100`.                                                                          |
+| disableAutomaticEventLogging | Boolean        | Disables logging of sdk generated events (e.g. aggVariableEvaluated, aggVariableDefaulted) to DevCycle.                                                                      |
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/java/java-gettingstarted.md:78]: boolean should not be capitalized

File: docs/sdk/server-side-sdks/php/php-usage.md (hunk 12 above)
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/php/php-usage.md:220]:
```suggestion

```

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 17]
```diff
@@ -80,3 +80,48 @@ on_worker_boot do
...
+The SDK exposes various initialization options which can be set when registering the DevCycle Client:
+
+```ruby
+# Load the gem
+require 'devcycle-ruby-server-sdk'
+
+# Setup authorization
+options = DevCycle::Options.new(enable_edge_db: true, enable_cloud_bucketing: true)
+
+devcycle_client = DevCycle::Client.new("dvc_server_token_hash", options, true)
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:95]: everywhere else you use an env pull - not hardcoding

**Review by JamieSinn — CHANGES_REQUESTED** [review]

File: docs/sdk/server-side-sdks/go/go-gettingstarted.md (hunk 14 above)
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:91]: these were there before I touched any of this stuff

Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:91]: Yeah, I'm asking if you want to clean up these docs or not

Comment (cdruxerman) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:91]: the scope of what I'm trying to do here has ballooned a lot. I'd like to stick to where this has gotten to and try to finally get it out and then come back later if there are things we want to clean up further

Comment (JamieSinn) [reply docs/sdk/server-side-sdks/go/go-gettingstarted.md:91]: kk

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 17 above)
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:95]: this isn't my change, we can do a follow-up change if you'd like to see this differently

Comment (JamieSinn) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:95]: The diff is attributing it to you; so not sure where else you're seeing this

File: docs/sdk/server-side-sdks/python/python-usage.md (hunk 13 above)
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/python/python-usage.md:194]: Bump - this is just incorrect.

File: docs/sdk/server-side-sdks/dotnet/dotnet-usage.md [hunk 18]
```diff
@@ -107,3 +107,30 @@ To further enable various deployment configurations - we provide a background pr
...
+```csharp
+    client = new DevCycleLocalClientBuilder()
+        .SetSDKKey("<DEVCYCLE_SERVER_SDK_KEY>")
+        .SetOptions(new DevCycleLocalOptions(enableBetaRealtimeUpdates: true, eventFlushIntervalMs: 60000))
+        .SetInitializedSubscriber((o, e) => {
+            if (e.Success) {
+                ClientInitialized();
+            } else {
+                Console.WriteLine($"Client did not initialize. Errors: {e.Errors}");
+            }
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/dotnet/dotnet-usage.md:132]:
```suggestion
            // ... 
```

File: docs/sdk/server-side-sdks/dotnet/dotnet-gettingstarted.md [hunk 19]
```diff
@@ -87,17 +84,63 @@ namespace Example {
...
+namespace Example {
+    public class Example {
+        private static DevCycleLocalClient client;
+
+        static async Task Main(string[] args) {
+            client = new DevCycleLocalClientBuilder()
+                .SetSDKKey("<DEVCYCLE_SERVER_SDK_KEY>")
+                .SetOptions(new DevCycleLocalOptions(configPollingIntervalMs: 60000, eventFlushIntervalMs: 60000))
+                .SetInitializedSubscriber((o, e) => {
+
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/dotnet/dotnet-gettingstarted.md:115]:
```suggestion
                    // ...
```

File: docs/sdk/server-side-sdks/php/php-gettingstarted.md [hunk 20]
```diff
@@ -37,3 +37,24 @@ try {
 }
 ```
 [//]: # (wizard-initialize-end)
+
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when creating a new devCycleClient:
```
Comment (JamieSinn) [inline docs/sdk/server-side-sdks/php/php-gettingstarted.md:43]:
```suggestion
The SDK exposes various initialization options which can be set when creating a new DevCycleClient:
```

File: docs/sdk/server-side-sdks/python/python-usage.md (hunk 13 above)
Comment (JamieSinn) [reply docs/sdk/server-side-sdks/python/python-usage.md:194]: Still incorrect

Comment (cdruxerman) [reply docs/sdk/server-side-sdks/python/python-usage.md:194]: sorry i missed this. fixed

**Review by JamieSinn — APPROVED** [review]

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 21]
```diff
@@ -80,3 +80,48 @@ on_worker_boot do
...
+
+## Initialization Options
+
+The SDK exposes various initialization options which can be set when registering the DevCycle Client:
+
+```ruby
+# Load the gem
+require 'devcycle-ruby-server-sdk'
+
+# Setup authorization
```
Comment (ajwootto) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:92]: what does this comment mean?

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md [hunk 22]
```diff
@@ -80,3 +80,48 @@ on_worker_boot do
...
+options = DevCycle::Options.new(enable_edge_db: true, enable_cloud_bucketing: true)
```
Comment (ajwootto) [inline docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:93]: should the basic example of how to use options really include a bunch of options that won't mean anything to the reader at this point?

File: docs/sdk/server-side-sdks/python/python-usage.md [hunk 23]
```diff
@@ -171,3 +171,26 @@ To further enable various deployment configurations - we provide a background pr
...
 See the [SDK Proxy](../../sdk-proxy/index.md) section for more information.
+
+## Enabling Beta Realtime Updates
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
```
Comment (ajwootto) [inline docs/sdk/server-side-sdks/python/python-usage.md:182]: wouldn't the more relevant user-facing benefit of this be that flag updates are delivered more quickly? This all sounds like technical detail that mostly we care about internally

File: docs/sdk/features.md [hunk 24]
```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
 
 ## Realtime Updates
 
-DevCycle SDKs are capable of being notified in real time that new configuration changes have been made in the DevCycle platform.
+DevCycle SDKs (both client-side and server-side) are capable of being notified in realtime that new configuration changes have been made in the DevCycle platform.
```
Comment (ajwootto) [inline docs/sdk/features.md:305]: do we need to clarify that it's both client and server side? why would anyone assume differently when reading this?

File: docs/sdk/features.md [hunk 25]
```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
...
 DevCycle leverages Server-Sent Events (SSE) to notify the SDKs that a feature (targeting rules, variable values, etc.)
-has been saved and that they should fetch the new configuration.
+has been saved and that they should fetch the new configuration. A connection URL is included in the config that the SDK fetches,
```
Comment (ajwootto) [inline docs/sdk/features.md:307]:
```suggestion
has been updated and that they should fetch the new configuration. A connection URL is included in the config that the SDK fetches, 
```

File: docs/sdk/features.md [hunk 26]
```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
 
 ## Realtime Updates
 
-DevCycle SDKs are capable of being notified in real time that new configuration changes have been made in the DevCycle platform.
+DevCycle SDKs (both client-side and server-side) are capable of being notified in realtime that new configuration changes have been made in the DevCycle platform.
 DevCycle leverages Server-Sent Events (SSE) to notify the SDKs that a feature (targeting rules, variable values, etc.)
-has been saved and that they should fetch the new configuration.
+has been saved and that they should fetch the new configuration. A connection URL is included in the config that the SDK fetches, 
+triggering the SDK to open a connection with our SSE provider listening for any changes in the given environment.
```
Comment (ajwootto) [inline docs/sdk/features.md:308]:
```suggestion
triggering the SDK to open a connection with our SSE provider and listen for any changes in the given environment.
```

File: docs/sdk/features.md [hunk 27]
```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
...
+
+**Supported SDKs**
+- Javascript
+- React
+- iOS
+- Android
+- Flutter
+- NodeJS (including Next and Nest)
```
Comment (ajwootto) [inline docs/sdk/features.md:316]: Next is a different SDK and is not related to the Node SDK

File: docs/sdk/features.md [hunk 28]
```diff
@@ -302,30 +302,39 @@ For instructions on setting up a custom domain, see [Custom Domains](/platform/c
...
+#### **iOS SDK**, **Android SDK** & **Flutter SDK**
 
-##### **Javascript SDK**, **React SDK**
+If the user backgrounds the application for some period of time, the SDK will disconnect from the SSE provider and will reconnect again when the user brings the application to the foreground. When the application is brought to the foreground the SDK will request a new configuration to receive any updates it may have missed while the realtime connection was closed.
 
-If the user loses focus on the webpage for longer then the `inactivityDelay` (the default of which is set to 2 minutes, and can be configured through the options), the SDK will disconnect from the SSE provider and will reconnect when the user opens the tab / window again (i.e. the page's visibility state = `visible`). The SDK will also request a new configuration during reconnection to receive any updates it may have missed while the realtime connection was closed.
+#### Server-Side SDKs
 
-##### **iOS SDK**, **Android SDK** & **Flutter SDK**
+If the server loses its connection it will be re-opened on the next config poll, which can be configured as desired.
```
Comment (ajwootto) [inline docs/sdk/features.md:336]: what would "next config poll" mean to someone reading this?

File: docs/sdk/features.md (hunk 24 above)
Comment (cdruxerman) [reply docs/sdk/features.md:305]: @ajwootto it feels like there's no harm in being explicit, especially given we didn't start from a place of obviously doing all of the sdks

File: docs/sdk/server-side-sdks/python/python-usage.md (hunk 23 above)
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/python/python-usage.md:182]: @ajwootto polling every second means the speed of realtime updates is less relevant on server-side. The biggest benefit server-side is I can dial-down polling while still getting updates delivered quickly.

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 21 above)
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:92]: removed it

File: docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md (hunk 22 above)
Comment (cdruxerman) [reply docs/sdk/server-side-sdks/ruby/ruby-gettingstarted.md:93]: swapped the basic example to be what we want, enable realtime updates, make config poll longer

File: docs/sdk/features.md (hunk 28 above)
Comment (cdruxerman) [reply docs/sdk/features.md:337]: given it's client side, is next initialization better?

Comment (cdruxerman) [reply docs/sdk/features.md:337]: ok edited

**Pull request merged**